.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: scala
+
+scala:
+  - 2.12.5
+
+jobs:
+  include:
+    - stage: publishLocal
+      script: sbt ++$TRAVIS_SCALA_VERSION publishLocal
+    - stage: tests
```

Comment:
You do not need to repeat the `stage` name. It is possible to have consecutive `- script` entries which then uses previous stage name.